authors/apps/article/urls.py

```diff
@@ -13,5 +13,7 @@
     path('articles/<slug>/dislike', DislikeAPIView.as_view()),
     path('articles/<slug>/comments/', CommentsAPIView.as_view()),
     path('articles/<slug>/comments/<pk>', RetrieveCommentsAPIView.as_view()),
-    path('articles/<slug>/comments/<pk>/subcomment', SubCommentAPIView.as_view())
+    path('articles/<slug>/comments/<pk>/subcomment', SubCommentAPIView.as_view()),
+    path('articles/like_unlike_comment/<pk>', LikeUnlikeAPIView.as_view()),
```

Comment:
I also think we should rename this link to just like_comment/pk. the APIView name can remain as is to give more description though